artiq/coredevice/exceptions.py

```diff
@@ -174,3 +174,7 @@ class I2CError(Exception):
 class SPIError(Exception):
     """Raised when a SPI transaction fails."""
     artiq_builtin = True
+
+class UnwrapNoneError(Exception):
+    """Raised when unwrapping a none Option."""
+    artiq_builtin = True
```

Comment:
So, should I remove the `UnwrapNoneError` from the firmware of the release branches as its not really needed or used there and having it will cause some of the tests to fail due to missing definitions on the host side.